README.md

```diff
@@ -2,26 +2,34 @@
 Bowling Challenge
 =================
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+## Usage
 
-## The Task
+```
```

Comment:
Nice, clear instructions for interacting with the code. I couldn't clone as I didn't have access rights, so I copy-pasted straight into node